environments/yoma-ga/aca-py-agent.default.env

```diff
@@ -75,4 +75,6 @@ ACAPY_AUTO_RESPOND_PRESENTATION_PROPOSAL=false
 ACAPY_AUTO_RESPOND_PRESENTATION_REQUEST=false
 ACAPY_AUTO_STORE_CREDENTIAL=false
 ACAPY_AUTO_VERIFY_PRESENTATION=false
-ACAPY_PRESERVE_EXCHANGE_RECORDS=true
\ No newline at end of file
+ACAPY_PRESERVE_EXCHANGE_RECORDS=true
+
+AGENT_ENDPOINT=http://yoma-ga-agent:3020
```

Comment:
this is the important big. The AGENT_ENDPOINT is what an agent uses to provide connections to itself. This used to be generated by the ngrok startup. 

The ngrok wait script used to set this value so maybe to plugin and plugout ngrok we just enable/disable the ngrok wait script - by copying a blank or copying the real thing... and the real thing will override this env var...